src/ctim/examples/casebooks.cljc

```diff
@@ -1,11 +1,11 @@
 (ns ctim.examples.casebooks
   (:require [ctim.schemas.common :as c]
-            [ctim.examples.actors :refer [actor-maximal]]
             [ctim.examples.bundles :refer [bundle-maximal]]))
 
 (def observables [{:type "hostname" :value "Demo_Cta"}
                   {:type "amp_computer_guid" :value "68e94bf7-e239-4821-90d6-b7eaa0233443"}
                   {:type "ms_machine_id" :value "19de2b42b3ab0df8c15d117b7760ffa40da59d31"}
+                  {:type "trend_micro_id" :value "replace-with-better-example"}
```

Comment:
@ceilingduster do you have an example value that we could use here, please?

Reply:
54309955-1468-4dc8-b1bb-ddb79e1ca2a6